testapps/build_pipeline/app.yaml

```diff
@@ -0,0 +1,8 @@
+runtime: php
+env: flex
+
+runtime_config:
+  document_root: web
+
+env_variables:
+  WHITELIST_FUNCTIONS: phpinfo,exec
```

Comment:
Ah, I see. The runtime builder already implemented this, so we're making sure that we're using the builder.